src/ert/run_models/base_run_model.py

```diff
@@ -90,8 +101,9 @@ def __init__(
         self._job_start_time: int = 0
         self._job_stop_time: int = 0
         self._indeterminate: bool = False
-        self._fail_message: str = ""
-        self._failed: bool = False
+        self._failed = False
```

Comment:
why was typing removed?

Reply:
Not necessary. `False` has type `bool`.

Reply:
`self._failed: bool = False` ?

Reply:
We would like `mypy` to err if you later do `self._failed = 0`. Will mypy err like this if you remove `: bool`?